packages/ui/src/layout/wui-flex/index.ts

```diff
@@ -0,0 +1,68 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { styleMap } from 'lit/directives/style-map.js'
+import styles from './styles'
+import type { Color, BorderRadius, Spacing } from '../../utils/TypesUtil'
+
+@customElement('wui-flex')
+export class WuiFlex extends LitElement {
+  public static styles = [styles]
+
+  @property() public backgroundColor?: Color
+
+  @property() public borderRadius?: BorderRadius
+
+  @property() public flexDirection?: string
+
+  @property() public flexWrap?: string
+
+  @property() public flexBasis?: string
+
+  @property() public flexGrow?: string
+
+  @property() public flexShrink?: string
+
+  @property() public alignItems?: string
+
+  @property() public justifyContent?: string
+
+  @property() public columnGap?: Spacing
+
+  @property() public rowGap?: Spacing
+
+  @property() public gap?: Spacing
+
+  @property() public width?: string
+
+  @property() public height?: string
+
+  @property() public padding?: string
+
+  public render() {
+    const inlineStyles = {
```

Comment:
Too many options here, flex component should only be responsible for layout / flex functionality, so we need to get rid of: `backgroundColor`, `borderRadius`, `padding`, `width`, `height`